cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -224,10 +214,15 @@ instance FromJSON PartialNodeConfiguration where
       pncSnapshotInterval
         <- Last . fmap RequestedSnapshotInterval <$> v .:? "SnapshotInterval"
       pncExperimentalProtocolsEnabled <- fmap Last $ do
-        failOnRemovedField v "TestEnableDevelopmentNetworkProtocols"
-          "TestEnableDevelopmentNetworkProtocols has been renamed to ExperimentalProtocolsEnabled"
+        mValue <- v .:? "ExperimentalProtocolsEnabled"
+
+        mOldValue <- v .:? "TestEnableDevelopmentNetworkProtocols"
+
+        forM_ mOldValue $ \_ -> do
```

Comment:
why do we need `forM_` here? Can't we just use `when`?

Reply:
yeah `when (isJust mOldValue)` should be the same

Reply:
Modified to use `when`.